AmazonChimeSDKDemo/AmazonChimeSDKDemo/VideoModel.swift

```diff
@@ -347,29 +367,69 @@ class VideoModel: NSObject {
                 updatedSources[key] = remoteVideoSourceConfigurations[key]
             }
         }
+
+        var videoCount:Int = remoteVideoCountInCurrentPage
+        if videoCount < remoteVideoTileCountPerPage {
+            for (source, _) in pendingRemoteVideoSourceConfigurations {
+                if isAttendeeIdContentShare(attendeeId: source.attendeeId) {
```

Comment:
nit: not related to your change but seems like we should just store content share sources separately.